aether-odk-module/conf/pip/requirements.txt

```diff
@@ -70,11 +70,11 @@ pyOpenSSL==19.1.0
 python-dateutil==2.8.1
 python-json-logger==0.1.11
 pytz==2019.3
-pyxform==0.15.1
-redis==3.3.11
+pyxform==1.0.1
```

Comment:
I would expect a breaking change here :scream: 

you can revert this line and update the dependency in another PR.

Reply:
https://github.com/XLSForm/pyxform/releases/tag/v1.0.1